health/api/src/main/java/com/cerner/beadledom/health/api/DependenciesResource.java

```diff
@@ -54,15 +40,6 @@
   @Path("/{name}")
   Response getDependencyAvailabilityCheckHtml(@PathParam("name") String name);
 
-  @ApiOperation(value = "Availability Check for Dependency",
-      notes =
-          "Invokes the basic availability check on the given dependency. "
-              + "The response code will match the code returned by the dependency, and will be "
-              + "omitted from the JSON.",
-      response = HealthDependencyDto.class)
-  @ApiResponses(value = {
-      @ApiResponse(code = 503, message = "unhealthy", response = HealthDependencyDto.class),
-      @ApiResponse(code = 200, message = "healthy", response = HealthDependencyDto.class)})
   @io.swagger.annotations.ApiOperation(value = "Availability Check for Dependency",
```

Comment:
I think these can stop using the fully qualified names and be directly imported now.

Reply:
71cd328e